bom/pom.xml

```diff
@@ -55,6 +55,11 @@
         <artifactId>java-driver-query-builder</artifactId>
         <version>4.18.2-SNAPSHOT</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.cassandra</groupId>
+        <artifactId>java-driver-guava-shaded</artifactId>
+        <version>4.18.2-SNAPSHOT</version>
```

Comment:
So we're versioning the shaded JAR based on... the driver release, which means we need to release a new version with every driver release... whether we've upgraded Guava or not?  This seems... strange.